recipe/build.sh

```diff
@@ -5,5 +5,5 @@ set -eu
     --enable-static=no \
     --enable-shared=yes
```

Comment:
```suggestion
    --enable-shared=yes \
    --verbose
```
This might give more output?